### PR TITLE
Search aliases added for 404 and 500 pages

### PR DESCRIPTION
**Pull request by adamsilver**
Now when you use the search box, you can type in 404 to get the ‘page not found’ page as a result. And 500 to get the ‘There's a problem with the service’ pages.

### CODE REVIEW

File: src/patterns/page-not-found-pages/index.md.njk [hunk 1]
```diff
@@ -3,7 +3,7 @@ title: Page not found pages
 description: A page not found tells someone we cannot find the page they were trying to view. They are also known as 404 pages.
 section: Patterns
 theme: Pages
-aliases:
+aliases: '404'
```
Comment (joelanman) [inline src/patterns/page-not-found-pages/index.md.njk]: looks like our other aliases don't have quote marks, are they needed here?

Comment (adamsilver) [reply src/patterns/page-not-found-pages/index.md.njk]: Yep. It didn't work without.

File: package-lock.json [hunk 2]
```diff
@@ -16301,4 +16301,4 @@
       "dev": true
     }
   }
-}
+}
```
Comment (joelanman) [inline package-lock.json]: not sure this is an intended change?

Comment (adamsilver) [reply package-lock.json]: No. Please help me, @joelanman — I commited the file by accident and attempted to revert. :(

**Pull request closed without merging**